### DISCUSSION TITLE
Integrating with log4j2

### DISCUSSION THREAD

**Discussion by noamlevy81** [Netflix/conductor#2309] [Q&A]
The documentation states that log4j properties can be passed through LOG4J_PROP, I couldn't find any use of that variable.
is there a way to pass log4j properties file to the server?

**Comment by kishorebanala** [comment]
Hey @noamlevy81 , Do you mean this? https://github.com/Netflix/conductor/blob/d32ad1622740221398b48589a44079febbcdc5fb/docs/docs/metrics/server.md#collecting-metrics-with-log4j

**Comment by aravindanr** [comment]
Conductor 3.0 does not use the LOG4J_PROP variable, but it supports all the features of [Spring Boot logging](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.logging)

**Reply by maheshyaddanapudi** [reply]
Just to add on to what @aravindanr mentioned.

This is the place where you can edit thelog4j properties if needed.

https://github.com/Netflix/conductor/blob/main/server/src/main/resources/log4j2.xml

Thanks and Regards,
Mahi.
